backend/app/assets/javascripts/spree/backend/global/animate_css.es6

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
+
+//
+// Handle clearing out of animation styles from complete animation.
+const animateCSS = (element, animation, speed, prefix = 'animate__') =>
+  new Promise((resolve) => {
+    const animationName = `${prefix}${animation}`
+    const node = document.querySelector(element)
```

Comment:
In my opinion `node` and `element` would be the variables of the same type.

I could see a code like this written:

```js
const element = document.querySelector(....)
```

That being said, I'd rename `element` variable to `selector` or `elementSelector` (I prefer longer, more explicit variable names)